### ISSUE TITLE
Baselines

### ISSUE THREAD

**Issue by devinkwok** [gsaltintas/lmc#25]
These experiments will most likely be left for rebuttal (low priority but important for completeness).

On the reference run only, same sweep of perturb_step and perturb_scale.

- [ ] Do different training times change barriers? Use different training times (1/2 shorter and 2x longer)
- [ ] Are butterfly effect experiments comparable to Frankle et al.'s LMC experiments? Use different loader seeds after perturb_step
- [ ] Do different layers affect barriers more/less? Do per-layer perturbations. Also plot LLFC per-layer cosine similarities of the interpolated networks
- [ ] What is the relative scale of training steps versus noise? At each perturb_step (no perturb_scale required), compute step mean and std for a sample of batches. Plot as baseline in L2 plots.
- [x] Does every hparam setting have a minimum noise at which there is no barrier? Do some specific weights matter more? than others? *Fixing #28 may also allow the usual experiments to find this with small enough perturb_scale.* Run standard setting, sample % of weights randomly or by gradient magnitude to perturb, vary % to reduce noise L2.
- [ ] Different initialization schemes have different sharpness - do our results hold on Xavier vs Kaiming vs muP?

On small-scale finetuning runs.

- [ ] Are larger/smaller barriers relative to normal training due to finetuning hparam settings? Use the same task between training and finetuning (e.g. CIFAR-10), or conversely, use regular training hparams to do finetuning (just change the task at perturb_step, but keep LR schedule the same)
- [ ] Are barriers only due to the pretraining weight distribution or is it task dependent? Pretrain on CIFAR-10 and finetune on a random task, or conversely, pretrain on a random task and finetune on CIFAR-10.
